packages/xstate-wallet/src/event-types.ts

```diff
@@ -0,0 +1,50 @@
+import {SimpleAllocation, Participant} from './store/types';
```

Comment:
I'm wondering if we should silo these to workflow. IE: Create an `application` folder containing `workflow.ts` and `events.ts`?

Reply:
Right now, I'm thinking of these events as "everything that is passed out of the messaging service". The plan is to add events like `ENABLE_ETHEREUM` to this collection. I thought by putting them at the top level, we'd be able to collect all of these events together. Given that there aren't very many of them, I think this should be ok for now, but prepared to revisit in the future. I think it would probably(?) make sense to scope UI events by protocol (but then again, there won't be very many of these either, so it might be just as easy to have all events in one place).